src/main/java/nextstep/courses/domain/image/Image.java

```diff
@@ -0,0 +1,38 @@
+package nextstep.courses.domain.image;
+
+public class Image {
+    private final int capacity;
+    private final ImageType imageType;
+    private final double width;
+
+    private final double height;
+    private final double ratio;
+
+    public Image(int capacity, ImageType imageType, int width, int height) {
+        if (capacity < 1) {
```

Comment:
일반적으로 용량은 byte 단위로 표기 및 계산합니다.
매직 넘버 1이 어떠한 단위를 가졌는지 알기 어려우니 상수로 관리하면 어떨까요?
300, 200, 1.5 모두 마찬가지 입니다 

그리고 비교가 잘못된 것 같아요.. 1보다 작으면 예외를 던지면 안되지 않을까요?
```suggestion
        if (capacity > 1) {
```